libsodium-sys/src/crypto_generichash.rs

```diff
@@ -70,3 +94,55 @@ fn test_crypto_generichash_primitive() {
         assert_eq!(s, crypto_generichash_PRIMITIVE.as_bytes());
     }
 }
+
+#[test]
+fn test_crypto_generichash_statebytes() {
+    assert!(unsafe { crypto_generichash_statebytes() } > 0);
+}
+
+#[test]
```

Comment:
The tests in libsodium-sys shouldn't be tests of the actual primitives, instead it should be tests of the bindings, e.g. that all constants are correct and so on. All test vectors of actual primitives should be defined in `crypto::generichash`. I would like to see (at least) all test vectors from `libsodium`.